srcpkgs/docker-cli/template

```diff
@@ -2,7 +2,7 @@
 # should be kept in sync with moby
 pkgname=docker-cli
 version=24.0.6
-revision=1
+revision=2
```

Comment:
Looks like you bumped the wrong package; it should be `moby` that gets bumped.

Reply:
Oops, thanks. Will do.

Reply:
done